### ISSUE TITLE
No image in Termux-x11

### ISSUE THREAD

**Issue by TheLittleYellowFiat** [Herick75/Box4Droid#32]
I followed the tutorial to install and start Box4Droid, and when I start the x11 server using `x11` I can see there is a connection established in the app, but the screen is blank (as is expected if I'm not wrong). Then, when I start Box86 using `start-box86`, it does not return any errors, but in the termux-x11 app the screen is still blank like it was before the `start-box86` command.

Device: POCO X3 PRO (vayu)
System: Lineage OS 18.1 (Android 11)
Version: 0.0.1 (I have tried 0.0.2 and exactly the same issue appears.)

**Comment by Garycolmann** [comment]
I'v been having the same exact issue. My current phone is a Samsung A22-5M/DSN, whit root and android 13 installed. Can we get a fix please?

**Comment by Garycolmann** [comment]
Btw, i'v tried to install the latest universal Termux-X11 from the original repo. The most recent one as of the date i'm writing this. Dont know if this has some implications...

**Comment by TheLittleYellowFiat** [comment]
Update: I tried this on another phone, a Redmi Note 9 Pro, also running lineage 18.1 (Android 11), and the results are exactly the same.

**Comment by Herick75** [comment]
Sorry for the problems, you may be updating Box4Droid to version 0.0.3, this version fixes some issues related to that, I suggest you take a look

**Comment by TheLittleYellowFiat** [comment]
Thanks for the update, unfortunately I still can not get it to work. ~~When I start Box4Droid using `box` and select a resolution (I used 1400x600) Termux:X11 shows the same blank screen with a cursor, and in Termux there is a menu to select between starting Box64, starting Winetricks GUI etc. When I try starting Box64, it says `Checking container integrity...` `Failed.` `Please check the Containers menu to see if the selected container exists or is corrupted.`, then a screen saying `No containers available.` is shown.~~ I managed to fix the no containers error by creating a new one and installing Wine using the provided options, now Box64 + Wine starts normally but now Termux:X11 shows no connection, same thing happens when I choose the second option from the menu, `Start Winetricks GUI`, in Termux there are no errors, except it saying it's discouraged to run Winetricks as root, but there is still no connection in Termux:X11

**Comment by Herick75** [comment]
Sorry for the issues, try installing the stable version of Box4Droid, now most issues are fixed

**Comment by TheLittleYellowFiat** [comment]
I installed the newest version again, it now works, thanks. (I needed to create a new container and install wine in it for it to work, but it was easy enough following the instructions on-screen, putting this here if anyone has to do the same). Currently I'm having issues running an .exe file like in issue #26, but I'll figure it out.
Thanks again, it's gonna be fun playing around with it, you really made a useful piece of software.
